test/GuardClauses.UnitTests/GuardAgainstExpression.cs

```diff
@@ -26,60 +26,49 @@ public static IEnumerable<object[]> GetCustomStruct()
     [InlineData(10)]
     public void GivenIntegerWhenTheExpressionEvaluatesToTrueDoesNothing(int test)
     {
-        Guard.Against.AgainstExpression((x) => x == 10, test, "Value is not equal to 10");
+        Guard.Against.Expression((x) => x == 10, test, "Value is not equal to 10");
     }
 
     [Theory]
     [InlineData(10)]
     public void GivenIntegerWhenTheExpressionEvaluatesToFalseThrowsException(int test)
     {
-        Assert.Throws<ArgumentException>(() => Guard.Against.AgainstExpression((x) => x == 5, test, "Value is not equal to 10"));
+        Assert.Throws<ArgumentException>(() => Guard.Against.Expression((x) => x == 5, test, "Value is not equal to 10"));
```

Comment:
Change this to a `[Fact]` since there is only one test case. The error message should say something like "Value cannot be 5" given that the expression is `x == 5` which, when true, will trigger the exception.